src/client/Client.cc

```diff
@@ -8235,9 +8235,11 @@ int Client::lookup_hash(inodeno_t ino, inodeno_t dirino, const char *name,
  * the resulting Inode object in one operation, so that caller
  * can safely assume inode will still be there after return.
  */
-int Client::lookup_ino(inodeno_t ino, const UserPerm& perms, Inode **inode)
+int Client::lookup_ino(inodeno_t ino, const UserPerm& perms, Inode **inode, bool lock)
```

Comment:
@jtlayton 
thanks for you reviews, updated it.
The initial purpose is not want to change the name of the current interface(lookup_ino/lookup_parent/lookup_name/ll_forget), because this needs to modify the function of other callers, but at the moment to see these few interface calls are not many changes easier